python/cugraph/graph/c_graph.pyx

```diff
@@ -139,7 +139,7 @@ class Graph:
 
         return cudf.Series(src_data), cudf.Series(dest_data)
 
-    def to_edge_list(self):
+    def view_edge_list(self):
```

Comment:
isn't this a duplicate function name from line 119

Reply:
Good catch, that function is duplicated. I will fix it. Thanks.